app/config/security.yml

```diff
@@ -1,53 +1,48 @@
-            # http://symfony.com/doc/current/cookbook/security/form_login_setup.html
-
-    encoders:
-        FOS\UserBundle\Model\UserInterface: bcrypt
-
-    role_hierarchy:
-        ROLE_ADMIN:       ROLE_USER
-        ROLE_SUPER_ADMIN: ROLE_ADMIN
-
-    providers:
-        fos_userbundle:
-            id: fos_user.user_provider.username
-
-    firewalls:
-        main:
-            pattern: ^/
-            form_login:
-                provider: fos_userbundle
-                csrf_token_generator: security.csrf.token_manager
-                # if you are using Symfony < 2.8, use the following config instead:
-                # csrf_provider: form.csrf_provider
-
-            logout:       true
-            anonymous:    true
-
-    access_control:
-        - { path: ^/login$, role: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/register, role: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/resetting, role: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/admin/, role: ROLE_ADMIN }
\ No newline at end of file
+security:
+    firewalls:
+        dev:
+            pattern:                    ^/(_(profiler|wdt)|css|images|js)/
+            security:                   false
+        main:
+            pattern:                    ^/
+            anonymous:                  true
+            form_login:
+                login_path:             fos_user_security_login
+                check_path:             fos_user_security_check
+                provider:               fos_userbundle
+                use_forward:            false
+                failure_path:           null
+                default_target_path:    /
+                remember_me:            true
+                csrf_token_generator:   security.csrf.token_manager
+            logout:
+                path:                   fos_user_security_logout
+                target:                 /login
+            remember_me:
+                key:                    %secret%
+                lifetime:               3000000000
+                path:                   /
+
+    encoders:
+        FOS\UserBundle\Model\UserInterface: bcrypt
+
+    role_hierarchy:
+        ROLE_ADMIN:       ROLE_USER
+        ROLE_SUPER_ADMIN: ROLE_ADMIN
+
+    providers:
+        fos_userbundle:
+            id: fos_user.user_provider.username_email
+
+    access_control:
+        - { path: ^/login$, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/register, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/resetting, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/create/, role: ROLE_USER }
+        - { path: ^/leave/, role: ROLE_USER }
+        - { path: ^/app_change_size/, role: ROLE_USER }
+        - { path: ^/user_profile/, role: ROLE_USER }
+        - { path: ^/dashboard/, role: ROLE_USER }
+        - { path: ^/components/, role: ROLE_USER }
+        - { path: ^/admin_profile/, role: ROLE_ADMIN }
+        - { path: ^/admin/, role: ROLE_ADMIN }

```

Comment:
Yes but you can set the security directly in the routing (via yaml or annotation)